.Rhistory

```diff
@@ -0,0 +1 @@
+getwd()
```

Comment:
Whoops, I just removed it. Thanks for catching that.

I opened one of the .md files I'd created for testing, and it opened in RStudio by default, causing the .Rhistory file to get created in the working directory.